tests/component/sections/dataset/useCheckPublishCompleted.spec.tsx

```diff
@@ -0,0 +1,75 @@
+/// <reference types="cypress" />
```

Comment:
Just curious on what type completions does this give us, these are really useful sometimes, I didn't know this one.

Reply:
Good question! I didn't notice that, I copied the test outline from another file and it was already there. That is meant for type completion of Cypress types like Command, Fixture, Chainable, etc.  I tried taking it out and file completion is still working for me in Webstorm, so maybe it depends on the IDE.